examples/jsm/shaders/SSRShader.js

```diff
@@ -163,7 +163,7 @@ var SSRShader = {
 			float totalStep=max(abs(xLen),abs(yLen));
 			float xSpan=xLen/totalStep;
 			float ySpan=yLen/totalStep;
-			for(float i=0.;i<MAX_STEP;i++){
+			for(float i=0.;i<float(MAX_STEP);i++){
```

Comment:
Will this convert `MAX_STEP` to a float in each loop and cause performance loss?

Reply:
i think that `float(MAX_STEP)` in cond-expr is a "compile-time evaluated" *constant*, so no.

Reply:
OK, thanks! And I have tested `MAX_STEP`(when is float already) and `float(MAX_STEP)`(float or int) really no fps difference, on my iphone6sp all 39~52fps, should be fine to keep the current state.

Would you mind change the other `Object.assign` order problem you found in `ReflectorForSSRPass.js`? I think it'll good  enought to report to maintainer after this change.

Reply:
@gonnavis PRed  #21385